test/box-luatest/gh_8605_add_box_info_hostname_test.lua

```diff
@@ -0,0 +1,18 @@
+local popen = require('popen')
+local t = require('luatest')
+
+local g = t.group()
+
+g.test_hostname_result = function()
+    local ph = popen.shell("hostname", 'r')
```

Comment:
Nit: `ph:close()` is desirable. It is collected by GC anyway, but the GC delay at freeing a tightly limited resource (such as a file descriptor) may cause unexpected effects under certain workloads. There may be a peak, where it hits the file descriptor limit.

Unlikely we'll be hit by this problem there, just a suggestion regarding the good practice.

Reply:
My bad, missed `close()`.

Reply:
Fixed.